web/src/main/java/org/springframework/security/web/authentication/switchuser/SwitchUserFilter.java

```diff
@@ -528,6 +529,15 @@ public void setUsernameParameter(String usernameParameter) {
 	}
 
 	/**
+	 * Allows the role of the switchAuthority to be customized.
+	 * 
+	 * @param switchAuthorityRole the role name. Defaults to {@link #ROLE_PREVIOUS_ADMINISTRATOR}
+	 */
+	public void setSwitchAuthorityRole(String switchAuthorityRole) {
```

Comment:
Not a problem.

As an FYI...We try to have an assertion for any setter on a property that has a default value. There may be some older setters that don't follow this convention. Also if the default value of the property is null, then there is no need for the assertion.